### ISSUE TITLE
False positive for unbalanced-tuple-unpacking

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#273]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

When using iterable unpacking:

```
#!python

a, b, *c = (1, 2, 3, 4, 5)
```

This will give:

```
#!python

W:  4, 0: Possible unbalanced tuple unpacking with sequence (1, 2, 3, 4, 5): left side has 3 label(s), right side has 5 value(s) (unbalanced-tuple-unpacking)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/273
